cli/attacks/signup.js

```diff
@@ -0,0 +1,46 @@
+const  ora = require('ora');
+const logSymbols= require('log-symbols');
+const { faker } = require('@faker-js/faker');
+const { http } = require('../lib/request');
+
+
+const threshold = 100;
+
+async function signups(targetURL, headers){
+
+    const spinner = ora('Starting user signups...').start();
+    
+    for (let i = 0; i < 11; i++) {
```

Comment:
We should use the `threshold` variable here and in the message below rather than the hardcoded constant `11`.